style.css

```diff
@@ -0,0 +1,27 @@
+h1{
+    color:blue
```

Comment:
Try to end your rules with a semicolon:
```suggestion
    color:blue;
```